docs/contributing/index.md

```diff
@@ -170,16 +170,16 @@ This section describes how to make contributions to files in the `plone/document
 1.  Members who subscribe to the repository will receive a notification and review your request.
 
 
-(contributing-editing-volto-documentation-label)=
+(contributing-editing-external-package-documentation-label)=
 
-### Editing Volto documentation
+### Editing external package documentation
```

Comment:
Same for other mentionings of external package Volto:
Rplace "Volto" by "external package"

![grafik](https://user-images.githubusercontent.com/1144737/159341487-6a772b11-0482-4978-949a-0627cfe02bec.png)

Reply:
Done in 5f93f5533ca5d77925e937bc71863476258c23cc